apis/hubspot/crm/companies/context.gpt

```diff
@@ -0,0 +1,5 @@
+name: context
+tools: sys.http.html2text
```

Comment:
Does this need to be `share tools` instead? I forget but want to make sure.

Reply:
I don't think so because I just want this context tool to use it, I don't want to give that to the end user